FirstTask.py

```diff
@@ -0,0 +1,180 @@
+from os import system as sys
+
+class BankAccount:
+    def __init__(self, Name, PassCode, AccNumber, Balance, AccoutType):
+        self.Name = Name
+        self.Passcode = PassCode
+        self.Balance = Balance
+        self.AccountType = AccoutType
+        self.AccNumber = AccNumber
+
+    def FullName(self):
+        return self.Name
+
+    def AccountNumber(self):
+        return self.AccNumber
+
+    def PassCode(self):
+        return self.Passcode
+
+    def CheckBalance(self):
+        return self.Balance
+
+    def CheckAccountType(self):
+        return self.AccountType
+
+    def AccDetails(self):
+        print("Account # ", self.AccNumber)
+        print("Name: ", self.Name)
+        print("Balance: ", self.Balance)
+        print("Account Type: ", self.AccountType)
+
+    def DepositBill(self,Bill):
+        if Bill >= 0:
+            self.Balance += Bill
+            return True
+        else:
+            return False
+
+    def DrawMoney(self,Money):
+        if self.Balance >= Money:
+            self.Balance -= Money
+            return True
+        else:
+            return False
+    
+    def TransferMoney(self,Amount):
+        if self.Balance >= Amount:
+            self.Balance -= Amount
+            return True
+        else:
+            return False
+
+    @staticmethod
+    def Login(obj, AccNo, Code):
+        while(1):
+            chk1 = False
+            for i in range(len(obj)):
+                if (AccNo == obj[i].AccountNumber() and Code == obj[i].PassCode()):
+                    return (True,obj[i])
+            print("\nInvalid Credentials\n")
+            input("\nPress Any Key to Continue!\n")
+            sys('clear')
+            return (False,obj[i])

```

Comment:
Parentheses `( )` are not required here.
If there is no match then why return `obj[i]`? You can send a message in its place that can be printed out if `False` from where it is called.
```suggestion
            return False, `Invalid credentials!`
```